modules/abstract-eth/src/lib/transferBuilder.ts

```diff
@@ -289,4 +295,17 @@ export class TransferBuilder {
       this._tokenContractAddress = transferData.tokenContractAddress;
     }
   }
+
+  public buildUnsignedTransaction(): Buffer<ArrayBuffer> {
```

Comment:
```suggestion
  public getSignatureData(): Buffer<ArrayBuffer> {
```
this is not an unsigned txn

Reply:
Added Tests